console/console.properties

```diff
@@ -146,23 +146,22 @@ orgTypeValues=Association,Company,NGO,Individual,Other
 # Areas map configuration
 # This map appears on the /console/account/new page, when the user checks the "my org does not exist" checkbox.
 # Currently the map is configured with the EPSG:4326 SRS.
-
 # Center of map
-AreaMapCenter=2.5, 50
+AreaMapCenter=9.3707, 42.0753
 
 # Zoom of map
-AreaMapZoom=6
+AreaMapZoom=7
 
 # AreasUrl is the URL of a static file or a service with a GeoJSON FeatureCollection object string in EPSG:4326.
 # example "dynamic" AreasUrl=https://my.server.org/geoserver/ows?SERVICE=WFS&REQUEST=GetFeature&typeName=gadm:gadm_for_countries&outputFormat=json&srs=EPSG:4326&cql_filter=ISO='FRA' or ISO='BEL'
```

Comment:
Missing a comment explaining that the file will also be searched for in the `<datadir>/console/` folder